osquery/tables/networking/darwin/wifi.mm

```diff
@@ -99,10 +97,12 @@ void parseNetworks(const CFDictionaryRef& network, QueryData& results) {
     auto key = CFStringCreateWithCString(kCFAllocatorDefault, kv.second.c_str(),
                                          kCFStringEncodingUTF8);
     CFTypeRef value = nullptr;
-    if (key != nullptr && CFDictionaryGetValueIfPresent(network, key, &value)) {
-      r[kv.first] = extractNetworkProperties(value);
+    if (key != nullptr) {
```

Comment:
I am on clang-700.1.76 as well but on 10.10. Can't seem to get `make analyze` to complain. I might look more into it, but will also be more careful.